charts/latest/azurefile-csi-driver/templates/csi-azurefile-controller.yaml

```diff
@@ -190,6 +190,7 @@ spec:
           args:
             - "--v={{ .Values.controller.logLevel }}"
             - "--endpoint=$(CSI_ENDPOINT)"
+            - "--enable-azurefile-proxy={{ .Values.node.azurefileProxy.enabled }}"
```

Comment:
this is not necessary change for controller, just remove this config

Reply:
My original intention was to use this parameter to intercept requests for creating EiT-type volumes at the volume creation stage when the user hasn't enabled azurefile-proxy, rather than waiting until the NodeStageVolume stage to block them.

Reply:
How about change this parameter as `support-encrypt-in-transit` ?

Reply:
the name is fine, this parameter is not used in controller since controller does not mount any volume, so just remove this param